docs/Getting_Started.md

```diff
@@ -1,17 +1,57 @@
 # Getting Started
 
-There are many places to start start in ElectionGuard. If you're looking to develop see below, if you just want to research see the [guide](guide/Overview.md). 
+There are many ways to adopt or contribute to ElectionGuard. It is an open source platform and depends on an active community and ecosystem.
 
-## Latest
+The ElectionGuard SDK comprises three Github repos:
 
-These are the latest projects from ElectionGuard. If you're looking to contribute, these are the best places to start.
+* ElectionGuard Specification
+* ElectionGuard Python
+* ElectionGuard C++
 
-[`electionguard-python`](guide/Core_Component.md#python)
+Whenever possible and as a guiding principle, the evolution of the ElectionGuard SDK will be driven by the use cases and priorities of the community and in service of best-practice implementations of real-world, secret-ballot public elections
 
-[`electionguard-web-api`](guide/Implementations.md)
+!!! info "Ways to get involved and find out what's actively being worked on"
```

Comment:
I'm not a huge fan of the title of this box and it feels very wordy. 

## Want to get involved?

- Post questions or ideas on our discussion boards
- Join our weekly office hours
- Develop code in our open source repos (link to developer section)
- Contribute to the docs and spec (link to each)